tests/TestHarness/CMakeLists.txt

```
@@ -1,7 +1,13 @@
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/__init__.py ${CMAKE_CURRENT_BINARY_DIR}/__init__.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/testUtils.py ${CMAKE_CURRENT_BINARY_DIR}/testUtils.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/WalletMgr.py ${CMAKE_CURRENT_BINARY_DIR}/WalletMgr.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/Node.py ${CMAKE_CURRENT_BINARY_DIR}/Node.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/Cluster.py ${CMAKE_CURRENT_BINARY_DIR}/Cluster.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/TestHelper.py ${CMAKE_CURRENT_BINARY_DIR}/TestHelper.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/launch_transaction_generators.py ${CMAKE_CURRENT_BINARY_DIR}/launch_transaction_generators.py COPYONLY)
+configure_file(__init__.py __init__.py COPYONLY)
+configure_file(testUtils.py testUtils.py COPYONLY)
+configure_file(WalletMgr.py WalletMgr.py COPYONLY)
+configure_file(Node.py Node.py COPYONLY)
+configure_file(Cluster.py Cluster.py COPYONLY)
+configure_file(TestHelper.py TestHelper.py COPYONLY)
+configure_file(queries.py queries.py COPYONLY)
+configure_file(transactions.py transactions.py COPYONLY)
+configure_file(libc.py libc.py COPYONLY)
+configure_file(interfaces.py interfaces.py COPYONLY)
+configure_file(launch_transaction_generators.py launch_transaction_generators.py COPYONLY)
+configure_file(logging.py logging.py COPYONLY)
+configure_file(depresolver.py depresolver.py COPYONLY)
```

I like how this is simplified, less clutter. We can further simplify if `<output>` is the relative path `.`.

```suggestion
configure_file(__init__.py . COPYONLY)
configure_file(testUtils.py . COPYONLY)
configure_file(WalletMgr.py . COPYONLY)
configure_file(Node.py . COPYONLY)
configure_file(Cluster.py . COPYONLY)
configure_file(TestHelper.py . COPYONLY)
configure_file(queries.py . COPYONLY)
configure_file(transactions.py . COPYONLY)
configure_file(libc.py . COPYONLY)
configure_file(interfaces.py . COPYONLY)
configure_file(launch_transaction_generators.py . COPYONLY)
configure_file(logging.py . COPYONLY)
configure_file(depresolver.py . COPYONLY)
```